src/instructlab/configuration.py

```diff
@@ -749,7 +749,9 @@ def _expand_paths(content: dict | list):
 
 def _expand_value(value):
     if isinstance(value, str):
-        return utils.expand_path(value)
+        value = os.path.expanduser(value)
+        value = os.path.expandvars(value)
+        return value
```

Comment:
Can we use diff vars here? Having them all called `value` is not great

Reply:
This is the same code that's in utils as well.  Could do it with no vars.

```
return os.path.expandvars(os.path.expanduser(value))
```

Reply:
Consider it renamed